manifests/pip/bootstrap.pp

```diff
@@ -11,6 +11,7 @@
 class python::pip::bootstrap (
   Enum['pip', 'pip3'] $version            = 'pip',
   Variant[Boolean, String] $manage_python = false,
+  String $http_proxy                      = '',
```

Comment:
Missing doc about this new parameter and then update of `REFERENCE.md`.

Reply:
Done.